### DISCUSSION TITLE
How to and SENDGRID_API_KEY ?

### DISCUSSION THREAD

**Discussion by Rohitbels** [AntlerVC/firetable#721] [Support / Q&A] [answered]
How and where to  add SENDGRID_API_KEY ?

Secret Manager setup:

Secret Name: sendgrid

Secret Value:
SENDGRID_API_KEY

**Comment by shamsmosowi** [comment, marked as answer]
Hi @Rohitbels you need to create secret on your gcp project

1. [Go to the Secret Manager page](https://console.cloud.google.com/security/secret-manager)  in the console.
2. On the Secret Manager page, click Create Secret.
3. On the Create secret page, under Name, enter a name for the secret **sendgrid**.
4. in the Secret value field, enter a value for the secret (SENDGRID_API_KEY).
5. Leave the Regions section unchanged.
6. Click the Create secret button.
